### DISCUSSION TITLE
Support `InputTransformer` in `EventBridgeRule` event

### DISCUSSION THREAD

**Discussion by ljacobsson** [aws/serverless-application-model#2666] [Ideas]
**Description:**

The options when using CloudWatchEvent/EventBridgeRule as event source are quite limited. A very useful feature to use here would be [InputTransformer](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-inputtransformer.html#cfn-events-rule-inputtransformer-inputtemplate)

I've tried achieving the same result using `Input` and `InputPath`, but they aren't comparable.

The effect of not allowing this is that one either sends too much data to the lambda function or has to resort to `AWS::Events::Rule`

**Comment by ShreyaGangishetty** [comment]
@ljacobsson  Thanks for the feature request! We have passed this along to our product team for possible prioritization. Please +1 on the feature to help with prioritization.

**Comment by antoniy-qb** [comment]
+1

**Comment by ljacobsson** [comment]
@praneetap ping about what we just talked about ⏫

**Comment by amancevice** [comment]
this feature would be so useful to our team I'm considering moving some stacks over to Terraform to take advantage

**Comment by ConnorRobertson** [comment]
Support for `InputTransformer` in `EventBridgeRule` event will rollout in the coming weeks. [View the change here!](https://github.com/aws/serverless-application-model/pull/3283)
